content/events/hacktoberfest.adoc

```diff
@@ -196,6 +196,16 @@ a| We are moving plugin documentation from https://wiki.jenkins.io/ to GitHub,
 link:https://issues.jenkins.io/issues/?jql=labels%20in%20(docs-sig%2C%20documentation)%20AND%20labels%20%3D%20newbie-friendly%20AND%20status%20in%20(Open%2C%20%22In%20Progress%22%2C%20Reopened)[Jira newcomer-friendly issues],
 link:https://github.com/search?q=org%3Ajenkinsci+is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22+label%3Adocumentation&type=Issues[GitHub good first issues]
 
+| French translation
+| Git, French, optional: Java developement tools to run Jenkins or plugins from sources locally.
+|
+Improve coverage of French localization of Jenkins.
+
+  * Some proposal are listed in link:https://issues.jenkins.io/browse/JENKINS-66658[a dedicated Epic].
+  * A topic is opened in the link:https://community.jenkins.io/t/topic-traduction-fr-pour-hacktoberfest-2021-sentez-vous-libre-de-me-rejoindre/405[French category of discourse].
+  * Use and improve if needed the localization end user facing documentation.
```

Comment:
I have not find yet if there is documentation that explain how to use jenkins with languages for end users. I have found the developper one, but not the end user one. If you find it please tell me :)

Reply:
found.